src/views/DatasetManager/components/DatasetList/DatasetList.js

```diff
@@ -108,10 +98,11 @@ export const DatasetList = () => {
       };
       const result = await TwoActionsDialogService.openDialog(dialogProps);
       if (result === 2) {
-        // TODO deleteDataset(datasetId);
+        if (currentDataset?.id !== dataset?.id) selectedDatasetBeforeDelete.current = currentDataset;
```

Comment:
I'm not sure why we'd need to store the id of the selected dataset, does the "delete" button change the selected dataset?